tRPC/prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  // NOTE: When using postgresql, mysql or sqlserver, uncomment the @db.Text annotations in model Account below
+  // Further reading:
+  // https://next-auth.js.org/adapters/prisma#create-the-prisma-schema
+  // https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#string
```

Comment:
based on this comment, should we uncomment `// @db.Text` schema for these?

![image](https://user-images.githubusercontent.com/55318172/229362175-640f4bcf-3a71-4e1e-baca-f78774b7e1f1.png)

Reply:
yes my bad